extensions/sql-migration/src/api/azure.ts

```diff
@@ -26,6 +26,27 @@ export async function getSubscriptions(account: azdata.Account): Promise<Subscri
 	return subscriptions.subscriptions;
 }
 
+export interface Location {
+	displayName: string,
+	name: string,
+	regionalDisplayName: string,
+}
+
+export async function getLocations(account: azdata.Account, subscription: Subscription): Promise<Location[]> {
```

Comment:
This seems generic enough it should go in the  azure extension, it's not specific to migration at all. See getResourceGroups below for an example of what I would expect this to look like.